js/web/lib/wasm/jsep/webgpu/ops/bias-split-gelu.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {DataType, tensorTypeToWsglType} from '../../../wasm-common';
+import {TensorView} from '../../tensor';
+import {ComputeContext, GpuDataType, ProgramInfo, ProgramMetadata} from '../types';
+import {ShaderHelper} from './common';
+import {erfImpl} from './unary-op';
+
+const validateInputs = (inputs: readonly TensorView[]): void => {
+    if (inputs[0].dataType !== DataType.float) {
+        throw new Error('inputs should be float type');
+    }
+
+    if (inputs[0].dims.length !== 3) {
+        throw new Error('input should have 3 dimensions');
+    }
+
+    if (![2560, 5120, 10240].includes(inputs[0].dims[2])) {
+        throw new Error('hidden state should be 2560, 5120 or 10240');
+    }
+
+    if (inputs[1].dims.length !== 1) {
+        throw new Error('bias is expected to have 1 dimensions');
+    }
+
+    if (inputs[0].dims[2] !== inputs[1].dims[0]) {
+        throw new Error('last dimension of input and bias are not the same');
+    }
+};
+
+const createBiasSplitGeluProgramInfo =
+    (metadata: ProgramMetadata, inputs: readonly TensorView[]):
+        ProgramInfo => {
+        const input = inputs[0];
+        const outputShape = input.dims.slice();
+        outputShape[2] = outputShape[2] / 2;
+
+        const gridSize = outputShape[0] * outputShape[1];
+        const halfHiddenSize = outputShape[2];
+        const dataType = tensorTypeToWsglType(inputs[0].dataType);
+        const blockSize = halfHiddenSize / 256;
+        const executionCount = gridSize * 256;
+
+        const getShaderSource = (shaderHelper: ShaderHelper) => `
+  const M_SQRT2 = sqrt(2.0);
+  const TPB = 256;  // threads per block, copied from CUDA kernel
+
+  @group(0) @binding(0) var<storage, read> input : array<${dataType}>;
```

Comment:
please use `shaderHelper.declareVariables()`. let me know if you have any problems using IndicesHelper.

Reply:
well... actually for this shader maybe it is fine. I see in the shader no code is calculating incides, instead they are just using the flatten index (offset). then do not have to use the IndicesHelper